creds.go

```diff
@@ -46,21 +49,19 @@ func (creds *Credentials) String() string {
 	)
 }
 
-// Client is the entry point into all the functionality in this package. It
-// contains authentication credentials, and a service endpoint, which are
-// required for all HTTP operations.
+// Client provides access to all taskcluster HTTP APIs for a given service, on
+// a given API version, of a given deployed cluster.
 type Client struct {
+	// Version of client API, e.g. "v1"
+	Version string
+	// Service name, e.g. "aws-provisioner"
+	Service string
```

Comment:
could we call this ServiceName?  To be consistent with the terminology used elsewhere?

Reply:
Sure!